components/script/dom/bindings/trace.rs

```diff
@@ -143,6 +144,7 @@ unsafe_no_jsmanaged_fields!(&'static Encoding);
 
 unsafe_no_jsmanaged_fields!(RefCell<Decoder>);
 unsafe_no_jsmanaged_fields!(RefCell<Vec<u8>>);
+unsafe_no_jsmanaged_fields!(RefCell<TimeRangesContainer>);
```

Comment:
This isn't true; TimeRangesContainer is contains JS objects

Reply:
You probably want to use RootableVec

Reply:
@Manishearth Sorry, I am not sure how `RootableVec` would help here. [TimeRangesContainer](https://github.com/servo/servo/blob/f220a63a0f4d7757c2a5e7c9da1c2063d37bebd6/components/script/dom/timeranges.rs#L60) contains [TimeRange](https://github.com/servo/servo/blob/f220a63a0f4d7757c2a5e7c9da1c2063d37bebd6/components/script/dom/timeranges.rs#L17) objects, which IIUC are no JS objects (although they are contained by JS objects). The way I understand it [TimeRanges](https://github.com/servo/servo/blob/f220a63a0f4d7757c2a5e7c9da1c2063d37bebd6/components/script/dom/timeranges.rs#L128) (note the `s`) is the JS object here.

Reply:
oh I see.

Reply:
I think this line is unnecessary if you use DomRefCell instead of RefCell.

Reply:
That's correct. I just updated the code. Thanks.